packages/ripple-binary-codec/src/serdes/binary-serializer.ts

```diff
@@ -145,9 +149,12 @@ class BinarySerializer {
    *
    * @param value length encoded value to write to BytesList
    */
-  public writeLengthEncoded(value: SerializedType): void {
+  public writeLengthEncoded(
+    value: SerializedType,
+    isUnlModifyWorkaround = false,
+  ): void {
     const bytes = new BytesList()
-    value.toBytesSink(bytes)
+    if (!isUnlModifyWorkaround) value.toBytesSink(bytes)
```

Comment:
i'm surprised the linter lets us do this?

shouldn't allow omitting braces. w/e for this PR

however if think we should comment what this workaround is all about